src/shogun/clustering/KMeansBase.cpp

```diff
@@ -321,7 +321,7 @@ void KMeansBase::init()
 	SG_ADD(
 	    &use_kmeanspp, "kmeanspp", "Whether to use kmeans++",
 	    ParameterProperties::HYPER | ParameterProperties::SETTING);
-	SG_ADD(&mus, "mus", "Cluster centers", ParameterProperties::MODEL);
+	SG_ADD(&mus, "cluster_centers", "Cluster centers", ParameterProperties::MODEL);
```

Comment:
this is not going to work though, as a param is registered with the same name below... I think the idea is that this would become something like "cluster_centers_initial" and the user can put a matrix in that parameter as an inital guess of the cluster centres?